spec.md

```diff
@@ -320,6 +323,19 @@ how platforms might expose these values to their users.
 
 After implementing the first endpoint `GET /v2/catalog` documented [above](#catalog-management), the service broker will need to be registered with your platform to make your services and plans available to end users.
 
+### Deprecating services and plans
+
+While the names and plans of services are technically mutable, changing the
+names of these entities is not supported uniformly on all platforms.  In order
+to provide users with the best experience when a service or plan name changes,
+broker authors SHOULD use the `deprecated` field to indicate that a service or
+plan will be removed in the future and create a new service or plan with the
+new name.
+
+Platforms MAY not allow new instances of deprecated services or plans to be
+created and MAY not allow existing service instances to be updated onto a
+deprecated plan.
```

Comment:
What about allowing existing instances of the deprecated plan to be updated (say to update configuration parameters)?

Reply:
Good catch, adding now